### PR TITLE
Update service programming model documentation for the IGNORED messag…

### PR DESCRIPTION
**Pull request by montaro**
See also #3251

### CODE REVIEW

File: docs/event-bus/docs/03-03-service-programming-model.md [hunk 1]
```diff
@@ -71,7 +71,26 @@ The HTTP POST request payload is a JSON object:
...
+A message delivered to a subscriber is considered successfully consumed if the service's HTTP response status code is
+`2xx`, for example:
+```json
+{
+    "event-id": "22ae22a4-f5b7-4fa1-ada9-558a10a96f3d",
+    "status": "published",
+    "reason": "Message successfully published to the channel"
+}
+```
+If the status code is not `2xx` (< 200 or >= 300), it means that a message consumption is not successful and
```
Comment (bszwarc) [inline docs/event-bus/docs/03-03-service-programming-model.md]:
```suggestion
If the status code is not `2xx` (< `200` or >= `300`), it means that the message was not consumed successfully and
```

File: docs/event-bus/docs/03-03-service-programming-model.md [hunk 2]
```diff
@@ -71,7 +71,26 @@ The HTTP POST request payload is a JSON object:
...
+that the message delivery is re-tried. This implies **At-least-once** delivery guarantee.
```
Comment (bszwarc) [inline docs/event-bus/docs/03-03-service-programming-model.md]:
```suggestion
that the Event Bus will try to deliver it again. This implies **At-least-once** delivery guarantee.
```

File: docs/event-bus/docs/03-03-service-programming-model.md [hunk 3]
```diff
@@ -71,7 +71,26 @@ The HTTP POST request payload is a JSON object:
...
+Note: If there were no subscriptions or consumers to this `event-type`, the message will be ignored and the response
```
Comment (bszwarc) [inline docs/event-bus/docs/03-03-service-programming-model.md]: >**NOTE**:
```suggestion
>**NOTE**: If there were no subscriptions or consumers to this `event-type`, the message is ignored and the response
```

File: docs/event-bus/docs/03-03-service-programming-model.md [hunk 4]
```diff
@@ -71,7 +71,26 @@ The HTTP POST request payload is a JSON object:
...
+would be like:
```
Comment (bszwarc) [inline docs/event-bus/docs/03-03-service-programming-model.md]:
```suggestion
will look like this:
```

File: docs/event-bus/docs/03-03-service-programming-model.md (hunk 1 above)
Comment (montaro) [reply docs/event-bus/docs/03-03-service-programming-model.md]: OK

File: docs/event-bus/docs/03-03-service-programming-model.md (hunk 2 above)
Comment (montaro) [reply docs/event-bus/docs/03-03-service-programming-model.md]: OK

File: docs/event-bus/docs/03-03-service-programming-model.md (hunk 3 above)
Comment (montaro) [reply docs/event-bus/docs/03-03-service-programming-model.md]: OK

File: docs/event-bus/docs/03-03-service-programming-model.md (hunk 4 above)
Comment (montaro) [reply docs/event-bus/docs/03-03-service-programming-model.md]: OK

**Pull request merged**